packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-basic-sign-in/main/aspnet/integrationsteps.md

```diff
@@ -4,7 +4,7 @@ After you complete the steps in the [Load the Widget](/docs/guides/oie-embedded-
 
 <div class="half">
 
-![Displays an example Widget sign-in page](/img/oie-embedded-sdk/oie-embedded-widget-use-case-sign-in-screen.png)
+![Screenshot of basic Okta Sign-In Widget](/img/siw/okta-sign-in-javascript.png)
```

Comment:
Yeah, this is because there aren't any original Figma diagrams for these. I've just used an existing SiW screenshot that already existed on the site, but was better quality than what was originally here.